firebase_admin/_user_import.py

```diff
@@ -472,11 +472,16 @@ def standard_scrypt(cls, memory_cost, parallelization, block_size, derived_key_l
 
 
 class ErrorInfo:
-    """Represents an error encountered while importing an ``ImportUserRecord``."""
+    """Represents an error encountered while performing a batch operation such
+    as importing users or deleting multiple user accounts.
+    """
+    # TODO(rsgowman): This class used to be specific to importing users (hence
+    # it's home in _user_import.py). It's now also used by bulk deletion of
+    # users. Move this to a more common location.
 
     def __init__(self, error):
-        self._index = error['index']
-        self._reason = error['message']
+        self._index = error.get('index', 0)
```

Comment:
Defaulting to index 0 sounds risky? Does the backend actually ever report an error without index? Seems safer to error out in this case than trying to sweep it under the rug.

Reply:
Yeah, good point. Done.